pipelines/Jenkinsfile

```diff
@@ -0,0 +1,248 @@
+pipeline {
+  options {
+    disableConcurrentBuilds()
+    buildDiscarder(logRotator(numToKeepStr: '7', artifactNumToKeepStr: '7'))
+  }
+
+  environment {
+    CONTEXT_NAME    = "etendo"
+    BBDD_SID        = "etendo"
+    BBDD_PORT       = "5432"
+    BBDD_SYSTEMUSER = "postgres"
+    BBDD_SYSTEMPASS = "syspass"
+    BBDD_USER       = "tad"
+    BBDD_PASSWORD   = "tad"
+    NEXUS_USER      = credentials('nexus-admin-user')
+    NEXUS_PASSWORD  = credentials('nexus-admin-passwd')
+    GITHUB_USER     = "etendobot"
+    GITHUB_TOKEN    = credentials('github-read-package-token')
+    EMAIL_ADDRESS   = credentials('email_builds')
+
+    ETENDO_BASE_URL = "https://github.com/etendosoftware/etendo_base"
+
+    JAVA_HOME       = "/usr/lib/jvm/java-11-openjdk-amd64"
+
+    SUCCESS         = "SUCCESS"
+    FAILED          = "FAILED"
+    UNSTABLE        = "UNSTABLE"
+  }
+
+  agent {
+    kubernetes {
+        inheritFrom 'jenkins-node-pg'
+        defaultContainer 'jnlp'
+        yaml """
+apiVersion: v1
+kind: Pod
+metadata:
+  name: jenkins-node-pg-0
+  namespace: jenkins
+  labels:
+    app.kubernetes.io/name: jenkins-node-pg
+spec:
+  volumes:
+    - name: rsa-keys
+      configMap:
+        name: rsa-keys
+        defaultMode: 384
+    - name: docker-sock
+      hostPath:
+        path: /var/run/docker.sock
+        type: ''
+  containers:
+    - name: compiler
+      image: etendo/compiler_jenkins:1.0.6
+      ports:
+        - name: ssh
+          containerPort: 22
+          protocol: TCP
+        - name: visualvm
+          containerPort: 8000
+          protocol: TCP
+      resources:
+        requests:
+          cpu: 1072m
+          memory: 4000Mi
+        limits:
+          cpu: 2072m
+          memory: 6500Mi
+      volumeMounts:
+        - name: rsa-keys
+          mountPath: /root/.ssh/
+        - name: docker-sock
+          mountPath: /var/run/docker.sock
+      lifecycle:
+        postStart:
+          exec:
+            command:
+              - bash
+              - '-c'
+              - >-
+                chmod a+x /var/run/docker.sock && rm
+                /etc/apt/sources.list.d/pgdg.list || echo 0 && apt update && apt
+                install -y curl
+      terminationMessagePath: /dev/termination-log
+      terminationMessagePolicy: File
+      imagePullPolicy: IfNotPresent
+  restartPolicy: Always
+  terminationGracePeriodSeconds: 30
+  dnsPolicy: ClusterFirst
+  serviceAccountName: default
+  serviceAccount: default
+  securityContext: {}
+  """
+    }
+  }
+
+  stages {
+    stage ('Build Enviroment') {
+      steps {
+        container('compiler') {
+          script {
+            sh './utils/repo-update-scripts/build-update-github.sh com.etendoerp.docker pending "Starting build" $ACCESS_TOKEN $CORE_GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'
```

Comment:
@RomanMagnoli Review comments on these lines in the 'post action'.

Reply:
Fixed!